lib/floe/workflow/runner/kubernetes.rb

```diff
@@ -113,7 +104,7 @@ def pod_spec(name, image, env, secret = nil)
             :spec       => {
               :containers    => [
                 {
-                  :name  => container_name(image),
+                  :name  => image_name(image),
```

Comment:
This one was interesting to me - not sure why we couldn't also use the pod name here?  Just curious why this was specially called out.  Regardless, I updated the spec to call out this difference.

Reply:
Typically what I've seen on kubernetes the pod has the "unique" suffix and the container name in the pod is the simple image name.  The difference with kubernetes and docker/podman is that in kubernetes we're creating a pod which contains a container and on docker/podman we're creating the container directly there is no other layer above it.

Reply:
Probably `floe-hello-world` would be the most consistent (e.g. `1-ems-metrics-processor-968dbbd67-sg886` container name is `1-ems-metrics-processor`) unfortunately this is neither container_name nor image_name

Reply:
Sounds good - as discussed, let's merge and I'll get it in the follow up for the vairous weird docker names